tests/integration/test_s3.py

```diff
@@ -241,3 +243,51 @@ async def test_synapse_backup_delete(
     await list_backups_action.wait()
     assert list_backups_action.status == "completed"
     assert "backups" not in list_backups_action.results
+
+
+@pytest.mark.s3
+@pytest.mark.usefixtures("s3_media_bucket")
+async def test_synapse_enable_media(
+    model: Model,
+    synapse_app: Application,
+    get_unit_ips: typing.Callable[[str], typing.Awaitable[tuple[str, ...]]],
+    access_token: str,
+    s3_integrator_app_media: Application,
+    boto_s3_media_client: typing.Any,
+    s3_media_configuration: dict,
+):
+    """
+    arrange: build and deploy the Synapse charm. Create an user and get the access token
+        Deploy, configure and integrate with Synapse the media-integrator charm.
+    act:  try to check if a given email address is not already associated.
```

Comment:
Is this the act of the test?

Reply:
must have been copied from somewhere, have made this more accurate